.github/workflows/build.yml

```diff
@@ -120,6 +120,51 @@ jobs:
       - name: Run Tests
         run: npm run wdio-cht-form
 
+  k3de2e:
+    needs: build
+    name: k3de2e
```

Comment:
Yea, I'm not sure what to do here. 
I mainly wanted a suite that runs over k3d. I thought the integration suite (api + sentinel) is a good choice, mainly because I wanted to flush out most of the potential bottlenecks (scaling containers, following logs and etc), as these suites cover most of the serverside complexity. 
This is also in regards to your comment about whether we're fine with an 20+ minute build. The answer is: maybe not? Maybe it's just pointless to run integration over 2 architectures? Should we just change both integration suites to run over k3d? 
I think that would be my preferred option.

Reply:
That works for me, I can't think of a reason integration tests need to be ran outside of k3d but a second opinion might be good